bosh-cf-crossregion/manifests/cross.yml

```diff
@@ -0,0 +1,1391 @@
+# All certs which are used in this file are for testing.
+# You should use your certs to replace them for production.
+---
+name: cross-region
+director_uuid: REPLACE_WITH_DIRECTOR_ID
+
+releases:
+- {name: cf, version: 231}
+
+networks:
+- name: cf_private
+  type: manual
+  subnets:
+  - range: 10.0.16.0/20
+    gateway: 10.0.16.1
+    dns: [168.63.129.16]
+    reserved: ["10.0.16.2 - 10.0.16.3"]
+    static: ["10.0.16.4 - 10.0.16.200"]
+    cloud_properties:
+      virtual_network_name: primaryboshvnet-crp
+      subnet_name: CloudFoundry
```

Comment:
@bingosummer In this case since we only have one vnet and one subnet for CF, we do not need to provide a `REPLACE_WITH` it's simply hardcoded to `CloudFoundry` here and passed in as `"subnetNameForCloudFoundry": "CloudFoundry"` in the ARM template. If you think its necessary to parameterize it, I can update it.